browser/ui/webui/brave_wallet/panel_handler/wallet_panel_handler.cc

```diff
@@ -52,10 +54,13 @@ void WalletPanelHandler::AddEthereumChainApproved(const std::string& payload,
     auto* prefs = user_prefs::UserPrefs::Get(contents->GetBrowserContext());
     ListPrefUpdate update(prefs, kBraveWalletCustomNetworks);
     base::ListValue* list = update.Get();
+    DLOG(INFO) << payload;
```

Comment:
Is this leftover? If you want to keep it for future debugging, please use DVLOG

Reply:
removed